src/themes/index.ts

```diff
@@ -1,7 +1,11 @@
-import { theme } from './converted'
+import { theme, lightTheme } from './converted'
 
-export async function loadTheme(editor: typeof import('monaco-editor-core').editor) {
-    const themeName = 'vs-code-theme-converted';
-    editor.defineTheme(themeName, theme);
-    return themeName;
+export async function loadTheme(
+  editor: typeof import('monaco-editor-core').editor,
+  dark = true) {
```

Comment:
Is this param really necessary? How about returning an object
`{ dark: 'vs-code-theme-converted-dark', light: 'vs-code-theme-converted-light' }`

Reply:
Of course can do this, then you need to judge whether to use `light` or `dark` when using it in the downstream repo

Reply:
Yes. Cuz we only need to execute `loadTheme` once so that we cannot get dark and light for one execution.